ste/xfer-deleteBlobFS.go

```diff
@@ -51,7 +54,16 @@ func doDeleteHNSResource(jptm IJobPartTransferMgr) {
 	transferDone := func(err error) {
 		status := common.ETransferStatus.Success()
 		if err != nil {
-			status = common.ETransferStatus.Failed()
+			var respErr *azcore.ResponseError
```

Comment:
Do we only want to apply the ignore 404 to dfs removes? what about file and blob removes?

Reply:
Agreed with @gapra-msft, we should be consistent.

Reply:
It looks like for blob and file, we also check 403, should we do a similar thing here?

Reply:
Done in the latest commit